docs/Behaviour - Server Side.md

```diff
@@ -31,19 +31,17 @@ If an active Receiver becomes unable to handle Flow, it is REQUIRED to take the
 
 ### Properties
 
-The `/properties` endpoint shows properties of the Input. In particular, it MAY contain information parsed from Base EDID and Effective EDID if the Input supports EDID and the Node is capable to parse EDID binaries.
+The `/properties` endpoint shows properties of the Input. In particular, it MAY contain information parsed from Base EDID (if set) and Effective EDID if the Input supports EDID and the Node is capable to parse EDID binaries.
 
 ### Base EDID
 
-The initial state of Base EDID is such an EDID which this Input presents to the upstream counterpart if Active Constraints are empty.
-
-`GET /edid/base` and `PUT /edid/base` operations allow a client to download and replace the Base EDID respectively if it exists. `DELETE /edid/base` resets Base EDID to the initial state. If the Base EDID for the Input changes and it is associated with any Senders, then all of the Senders in question MUST update their versions (in registered mode this MUST update the registered resources).
+There is no Base EDID at the initial state. If Base EDID for an Input changes, then all Senders associated with this Input MUST update their versions (in registered mode this MUST update the registered resources).
 
 ### Effective EDID
 
-Effective EDID is such combination of Base EDID and Active Constraints of all Senders associated with this Input that the baseband signal from the Input can be transmitted by the Senders without breaking Active Constraints.
+Effective EDID is such EDID that the baseband signal from the Input can be transmitted by the Senders associated with this Input without breaking their Active Constraints.
```

Comment:
I<m not sure I understand what this mean. I liked the original "Effective EDID is such combination of Base EDID and Active Constraints of all Senders associated with this Input"

Reply:
I reverted it and added language about what happens if there's no Base EDID set needed for this combination.